pyspelling/__init__.py

```diff
@@ -196,6 +198,13 @@ def _walk_src(self, targets, flags, pipeline):
                     except Exception as e:
                         err = self.get_error(e)
                         yield [filters.SourceText('', f, '', '', err)]
+            if not found_something:
```

Comment:
This whole block should be deindented to the same level as `for target in targets`. Targets is all the sources for a given task, so we want to do the check once all the targets have been evaluated.

Reply:
Well that's embarrassing, have fixed and I adjusted the test cases so it will catch if a missing source file would cause it to fail. Thanks for the catch.

Reply:
No problems. Thanks for working with me to get it looking good :slightly_smiling_face: .